### ISSUE TITLE
test 

### ISSUE THREAD

**Issue by publicodes** [datagir/nosgestesclimat#1304]
quand j'arrive aux questions alimentation, je sélectionne mes 14 menus et après je n'ai pas la main pour poursuivre : pas de bouton où cliquer pour continuer (je suis sur google chrome)

Depuis la page : `https://nosgestesclimat.fr/simulateur/bilan`

> Ce ticket a été créé automatiquement par notre robot depuis notre [page de contribution](https://nosgestesclimat.fr/contribuer).

**Comment by laem** [comment]
Étrange...

**Comment by Clemog** [comment]
Effectivement, ce n'est pas un bug remonté souvent. On peut éventuellement essayer de regarder si c'est bloquant en regardant le nombre d'abandons sur cette page.
